lib/Service.php

```diff
@@ -189,6 +189,27 @@ function write($rootElementName, $value, $contextUri = null) {
 
     }
 
+    /**
+     * Maps the given plain-old-php-object (popo) to the given rootElementName within the given namespace.
```

Comment:
I am used to the term popo from symfony or propel, but I am also ok with using a more 'pattern-oriented' term like ValueObject. 

I guess both terms are only known to experienced php programmers, so we can decide whatever we like best.